app/lib/cms/sts/assume_role_web_identity.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module CMS
```

Comment:
I'd need some help on understanding project folders, why is this under `CMS`? Is not that for the developer portal? and why under the `lib` folder? This is not a library but part of the project, it seems to me this would be better placed at `services/aws`

Reply:
This is not a service. More of a helper class. I think path was chosen because `s3.rb` is already there.

My guess without checking git log is that `s3.rb` is already there for historic reasons. 

For me `app/lib` is the proper location. And I prefer `s3.rb` and this one to stay together for consistency. Now I guess `s3.rb` can also be moved along with any other related classes if there are such. I wouldn't make it mandatory for this PR though.